src/java/mapImageWriter/MapImageWriter.java

```diff
@@ -0,0 +1,192 @@
+package mapImageWriter;
+
+import importer.SCMapImporter;
+import importer.SaveImporter;
+import map.*;
+import util.ArgumentParser;
+import util.FileUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Random;
+
+public strictfp class MapImageWriter {
+
+    public static boolean DEBUG = false;
+
+    private Path inMapPath;
+    private SCMap map;
+    private boolean writeImages;
+    private int mapImageSize;
+    private String writeImagesPath;
+    private boolean writeLayer0;
+    private boolean writeLayer1;
+    private boolean writeLayer2;
+    private boolean writeLayer3;
+    private boolean writeLayer4;
+    private boolean writeLayer5;
+    private boolean writeLayer6;
+    private boolean writeLayer7;
+    private boolean writeLayer8;
+    private boolean writeLayerh;
+
+    private SymmetrySettings symmetrySettings;
+
+    public static void main(String[] args) throws IOException {
+
+        Locale.setDefault(Locale.US);
+        if (DEBUG) {
+            Path debugDir = Paths.get(".", "debug");
+            FileUtils.deleteRecursiveIfExists(debugDir);
+            Files.createDirectory(debugDir);
+        }
+
+        MapImageWriter mapImageWriter = new MapImageWriter();
+
+        mapImageWriter.interpretArguments(args);
+
+        System.out.println("Creating map image files at " + mapImageWriter.inMapPath);
+        mapImageWriter.importMap();
+        mapImageWriter.writeMapImages();
+    }
+
+    public void interpretArguments(String[] args) {
+        interpretArguments(ArgumentParser.parse(args));
+    }
+
+    private void interpretArguments(Map<String, String> arguments) {
+        if (arguments.containsKey("help")) {
+            System.out.println("map-transformer usage:\n" +
```

Comment:
This isnt the map transformer

Reply:
lol, I'll fix the populator labelling too then xD